docs/Troubleshooting/resolving-splitbrain.md

```diff
@@ -308,15 +308,312 @@ Healed gfid:b23dd8de-af03-4006-a803-96d8bc0df004.
 Number of healed entries: 3
 ```
 
+# 3.1 Resolution of GFID split-brain using gluster CLI
+GFID split-brains can also be resolved by the gluster command line using the same policies that are used to resolve data and metadata split-brains.
+
+## i) Selecting the bigger-file as source
+This method is useful for per file healing and where you can decided that the file with bigger size is to be considered as source.
+
+Run the following command to obtain the path of the file that is in split-brain:
+```console
+# gluster volume heal VOLNAME info split-brain
+```
+
+From the output, identify the files for which file operations performed from the client failed with input/output error.
+### Example :
+```console
+# gluster volume heal testvol info
+Brick 10.70.47.45:/bricks/brick2/b0
+/f5
+/ - Is in split-brain
+
+Status: Connected
+Number of entries: 2
+
+Brick 10.70.47.144:/bricks/brick2/b1
+/f5
+/ - Is in split-brain
+
+Status: Connected
+Number of entries: 2
+```
+> **Note**
+> Entries which are in GFID split-brain may not be shown as in split-brain by the heal info or heal info split-brain commands always.
```

Comment:
It is better to clarify that for entry split-brains, it is the parent directory which is shown as being in split-brain. So one might need to run `info split-brain` to get the dir names and then `heal info` to get the list of files under that dir which *might* be in split-brain (it could just be needing heal without split-brain).

Reply:
Done.